packages/cache/src/internal/tar.ts

```diff
@@ -133,15 +147,22 @@ export async function listTar(
   archivePath: string,
   compressionMethod: CompressionMethod
 ): Promise<void> {
-  // --d: Decompress.
+  // -d: Decompress.
+  // unzstd is equivalent to 'zstd -d'
   // --long=#: Enables long distance matching with # bits.
   // Maximum is 30 (1GB) on 32-bit OS and 31 (2GB) on 64-bit.
   // Using 30 here because we also support 32-bit self-hosted runners.
   function getCompressionProgram(): string[] {
```

Comment:
This function seems repeated in two places. Shall we define a common function and use in both places.

Reply:
Yes we can do that. Currently it is an internal function defined in three places. I'll make the changes 👍